lib/suseconnect_register.pm

```diff
@@ -108,6 +114,19 @@ sub is_module {
     return 1 if (grep { $addon eq $_ } qw(asmm contm lgm pcm tcm wsm idu ids));
 }
 
+# De-register the system from the SUSE Customer Center
+sub suseconnect_deregistration {
+    my ($product) = @_;
+    my $deregister_cmd = "SUSEConnect --de-register";
+
+    $deregister_cmd .= " --product $product" if $product;
+    assert_script_run($deregister_cmd, 60);
+    assert_script_run("SUSEConnect --cleanup");
+
+    # Log SUSEConnect status and repos
+    script_run("SUSEConnect --status-text");
```

Comment:
Better always use assert_script_run because this simple status call should never fail. If it does we have mosr likely a serious problem we don't want to miss :)